scss/_buttons.scss

```diff
@@ -357,31 +357,27 @@
     }
   }
 
-  .btn + .button_to,
-  .button_to + .btn,
-  .button_to + .button_to {
+  .btn + .btn-form,
+  .btn-form + .btn,
+  .btn-form + .btn-form {
```

Comment:
Renamed `button_to` which is a rails default for a helper to `btn-form` which is what the helper creates. `form.btn-form button`